### PR TITLE
Issue/migrate footer styles tailwind

### LINKED ISSUE #235: Migrate the Footer component to tailwind
# Issue

## ZenHub Issue

We need to remove all sass code for the Footer component and migrate it to tailwind.

Footer component:
https://github.com/Coding-Coach/coding-coach/blob/develop/src/components/footer/Footer.js

Design:
https://sketch.cloud/s/xVR5A

### PR DESCRIPTION
**Pull request by rachelfranklin1**
# Details
This PR aims to migrate our styling from scss to tailwind. 
## Description
This PR intends to update the footer component's styling to match the rest of site, which uses tailwind to write CSS inline and in a shorthand way. 

## Steps to QA
 and compare with the changes on this branch locally. Check for responsive styling also.

- [ ] Step 1: Head to codingcoach.io 
- [ ] Step 2: Run this branch locally 
- [ ] Step 3: Compare the two footers and ensure the stylings are mostly identical
- [ ] Step 4: Check for responsive styling also.

## Screenshots
smaller screens:
Screenshot 2019-04-02 22 37 21
desktop screens:
Screenshot 2019-04-02 22 36 44

## Issue
Closes #235

### CODE REVIEW

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -5,20 +5,22 @@ import Button from 'components/button/Button';
...
-function Footer({ onClickLegal, t }) {
+function Footer({ onClickLegal }) {
   return (
-    <footer className={styles.footerMain}>
-      <div className={styles.footerInner}>
-        <div className={styles.column}>{/* <DonateButton /> */}</div>
+    <footer className={'w-full border-t border-secondary-lightest'}>
+      <div
+        className={
+          'flex flex-col md:flex-row justify-center items-center sm:justify-end flex-wrap pt-4 pb-4 pr-8 pl-8 mr-16 ml-16 '
```
Comment (tureey) [inline src/components/footer/Footer.js]: Would be nice to write this in more than 1 line so we don't have more than 80 chars per line

File: src/components/footer/__tests__/__snapshots__/Footer.test.js.snap [hunk 2]
```diff
@@ -2,14 +2,11 @@
 
 exports[`renders the footer element 1`] = `
 <footer
-  class="footerMain"
+  class="w-full border-t border-secondary-lightest"
 >
   <div
-    class="footerInner"
+    class="flex flex-col md:flex-row justify-center items-center sm:justify-end flex-wrap pt-4 pb-4 pr-8 pl-8 mr-16 ml-16 "
```
Comment (tureey) [inline src/components/footer/__tests__/__snapshots__/Footer.test.js.snap]: Same here

File: src/components/footer/Footer.js [hunk 3]
```diff
@@ -5,20 +5,22 @@ import Button from 'components/button/Button';
 
 // @TODO: Bring it back when ready
 // import DonateButton from './DonateButton';
-import styles from './footer.module.scss';
 
-function Footer({ onClickLegal, t }) {
```
Comment (tureey) [inline src/components/footer/Footer.js]: Removed on purpose?

Comment (rachelfranklin1) [reply src/components/footer/Footer.js]: thanks @tureey I removed the `t` that was being passed in to the Footer function as it wasn't being called from anywhere.

File: src/components/footer/__tests__/__snapshots__/Footer.test.js.snap (hunk 2 above)
Comment (rachelfranklin1) [reply src/components/footer/__tests__/__snapshots__/Footer.test.js.snap]: thanks @tureey ! Just updated! :)

**Comment by rachelfranklin1** [comment]
YAY! Happy to hear it! I'll keep an eye out for a task similar to this level to contribute too in the future :)

**Comment by rachelfranklin1** [comment]
What are the next steps, to getting this merged? :)

**Comment by crysfel** [comment]
Thanks for contributing!! 🎉

**Pull request merged**